### PR TITLE
feat(curriculum): create CSS Colors Quiz

### LINKED ISSUE #56184: [HACKTOBERFEST 2024] - CSS Colors Quiz
> [!IMPORTANT]
> This issue is part of our 2024 Hacktoberfest initiative. If you are interested in working on this issue, refer to our [top-level tasks](https://github.com/freeCodeCamp/freeCodeCamp/issues/56170) to see if this issue is claimed, or to claim it if it is not.

This issue is to create the CSS Colors quiz.

The file you should be working on is found in `/curriculum/challenges/english/25-front-end-development/quiz-css-colors`.

**All quiz questions need to relate to one of these topics:**

- color theory
- named colours
- rgb colour model
- hsl colour model
- hex codes
- linear and radial gradients

> [!CAUTION]
> Questions which do not fall under these topics will be rejected.

You are welcome to include code blocks in your questions, but please keep them short (four lines or less).

Please do NOT open a pull request until you have completed all questions. Your quiz should have **exactly 20 questions**.

### PR DESCRIPTION
**Pull request by pragyananda**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56184

### CODE REVIEW

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md [hunk 1]
```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`background: red;`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you set the transparency of a color to 50%?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md:64]:
```suggestion
How can you set the transparency of an element to 50%?
```

Colour transparency is managed by the alpha channel, the `opacity` property affects the entire element(s) targeted by the selector.

File: curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md [hunk 2]
```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`2`
 
 #### --answer--
 
-Placeholder answer
+`3`
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md:190]: These answers are not code and do not need to be in backticks.

File: curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md [hunk 3]
```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+`#ff0000`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which keyword sets the text color to the default browser color?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md:240]: Inherit does not set the value to the browser default. It sets the value to the computed value of the parent element. Which can sometimes be the default value, but not always.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
A few minor fixes.

File: curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md [hunk 4]
```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
...
 
 ---
 
-Placeholder distractor 3
+`color: any();`
 
 #### --answer--
 
-Placeholder answer
+No direct way to set.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md:124]:
```suggestion
There is no direct way to do so.
```

File: curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md [hunk 5]
```diff
@@ -17,439 +17,487 @@ Answer all of the questions below correctly to pass the quiz.
...
 
-Placeholder answer
+3
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of these functions allows specifying the saturation of a color?
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md:196]:
```suggestion
Which of these functions allows specifying the transparency of a color?
```

Saturation is the second value of `hsl`.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Last one!

**Review by naomi-lgbt — APPROVED** [review]
Great work!

**Comment by pragyananda** [comment]
hello @Sembauke can you please review my PR?

**Review by Sembauke — APPROVED** [review]

**Comment by pragyananda** [comment]
Thank you @Sembauke .

**Pull request merged**